### PR TITLE
Tightening up stats and also counting primary/rtx/padding separately

### PR DESCRIPTION
**Pull request by boks1971**

### CODE REVIEW

File: pkg/sfu/downtrack.go [hunk 1]
```diff
@@ -583,14 +603,37 @@ func (d *DownTrack) CreateSenderReport() *rtcp.SenderReport {
...
+	primaryStats, _ := d.primaryStats.Load().(*PacketStats)
+
+	primaryStats.octets += packetLen
+	primaryStats.packets += 1
+
+	d.primaryStats.Store(primaryStats)
+}
+
+func (d *DownTrack) UpdateRtxStats(packetLen uint32) {
+	rtxStats, _ := d.rtxStats.Load().(*PacketStats)
```
Comment (cnderrauber) [inline pkg/sfu/downtrack.go:616]: if need a accuracy stat here, will not get correct values when called concurrency, need use atomic.int in each single field

File: pkg/sfu/downtrack.go [hunk 2]
```diff
@@ -99,8 +105,10 @@ type DownTrack struct {
 	closeOnce               sync.Once
 
 	// Report helpers
-	octetCount   atomicUint32
-	packetCount  atomicUint32
+	primaryStats atomic.Value
```
Comment (cnderrauber) [inline pkg/sfu/downtrack.go:108]: better to add a comment about type of Value stored

**Comment by cnderrauber** [comment]
nice work!

File: pkg/sfu/downtrack.go (hunk 2 above)
Comment (boks1971) [reply pkg/sfu/downtrack.go:108]: Will do.

File: pkg/sfu/downtrack.go (hunk 1 above)
Comment (boks1971) [reply pkg/sfu/downtrack.go:616]: Yeah, can be tightened even further, but this is not that important to be super accurate. So, left it simpler for now.

**Review by cnderrauber — APPROVED** [review]

**Pull request merged**